test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -418,6 +450,31 @@ def test_create_a_payment_intent_and_void
     assert_equal 'requested_by_customer', cancel_response.params['cancellation_reason']
   end
 
+  def test_create_a_payment_intent_and_void_requires_unique_idempotency_key
+    idempotency_key = SecureRandom.hex
```

Comment:
Tiniest nitpick, but it might be worth using only `SecureRandom.hex` or `SecureRandom.uuid` when setting `idempotency_key` in this test file. I'm guessing it doesn't matter, but a future developer might wonder why different methods were used. What do you think?

Reply:
The use of `SecureRandom.hex` originated from a similar Adyen test. Ultimately it doesn't matter because both are generating random characters. I do agree we should standardize within this test to avoid confusion.  `SecureRandom.hex` is used more heavily in AM fwiw.